app/services/discover/register.rb

```diff
@@ -0,0 +1,34 @@
+module Discover
+  class Register
+    FOREM_DISCOVER_URL = "https://discover.forem.com/api/forems/register".freeze
+
+    def initialize(domain:)
+      @domain = domain
+    end
+
+    def self.call(**args)
+      new(**args).call
+    end
+
+    def call
+      return unless @domain
+
+      response = HTTParty.post(FOREM_DISCOVER_URL, body: { domain: @domain })
+
+      unless response.success?
+        error_message = %(
+          "Discover::Register Error - Forem Discover registration error.
+          #{response.message}.
+          #{response.body}."
+        )
+        Rails.logger.error error_message
```

Comment:
To be consistent with the rest of our codebase (according to a quick grep we have zero instances of `error` with parenthesis and 1 for `info`):

```suggestion
        Rails.logger.error(error_message)
```